Egregious-Eggnog/reactive-nutshell
# Sh articles post delete

# Description
Articles post new, delete existing and update existing. Forms for new and update

Fixes # (issue)

#9 #16 #21 #23

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout sh-articles-post-delete

Check that you are able to post new, delete and update articles. 
Harded coded as user 2 so there may be limited articles to view - please modify your json file or local storage if you do not see any articles.

Please check database.json to ensure that fields are properly populated.

# Checklist:
